<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trade</class>
 <widget class="QWidget" name="trade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/prefix1/image/trading.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="sheap">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/sheap.png</normaloff>:/prefix1/image/sheap.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="wheat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/wheat1.png</normaloff>:/prefix1/image/wheat1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="wood">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/wood1.png</normaloff>:/prefix1/image/wood1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="stone">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/stone1.png</normaloff>:/prefix1/image/stone1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="brick">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/brick2.png</normaloff>:/prefix1/image/brick2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="sheap_">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/sheap.png</normaloff>:/prefix1/image/sheap.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="wheat_">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/wheat1.png</normaloff>:/prefix1/image/wheat1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="wood_">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/wood1.png</normaloff>:/prefix1/image/wood1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="stone_">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/stone1.png</normaloff>:/prefix1/image/stone1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="brick_">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/brick2.png</normaloff>:/prefix1/image/brick2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="trade_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT</family>
     <pointsize>24</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Trade</string>
   </property>
  </widget>
  <widget class="QComboBox" name="trade_with">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tarde_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/prefix1/image/trade1.png</normaloff>:/prefix1/image/trade1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="sheap_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wheat_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wood_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="brick_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sheap_label_">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wood_label_">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="brick_label_">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wheat_label_">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone_label_">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;</string>
   </property>
   <property name="text">
    <string>Your Cards</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;</string>
   </property>
   <property name="text">
    <string>Their Cards</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
